### DISCUSSION TITLE
How to use liability correctly?

### DISCUSSION THREAD

**Discussion by M1ck0** [firefly-iii/firefly-iii#9387] [Firefly III]
I started using Firefly III today and I like it.

I have a question about how to use the liability feature. I have mortgage on the house with loan taken from bank. I added the data (image below).

How do I add money towards the loan? Do I add simple expense or something else?

Adding simple expense where the destination is this loan makes the loan goes from 118500 to 180000 which should not be the case since there is also an interest that should be calculated in final number (it should be 118308 instead of 180000).

Hopefully I explained this correctly. If you need more info tell me.

<img width="568" alt="Screenshot 2024-10-21 at 19 42 13" src="https://github.com/user-attachments/assets/db76604e-b433-4563-8609-3761798804bc">
<img width="902" alt="Screenshot 2024-10-21 at 19 51 21" src="https://github.com/user-attachments/assets/75543fde-09bb-48ef-9e3a-c4d50f7ecf76">

**Comment by JC5** [comment]
Firefly III doesn't calculate the interest for you. So that's something you must do yourself.

As for the expense, what do you want to achieve?

**Reply by M1ck0** [reply]
I was just giving expense as a example.

How do I make payment towards that loan? Do I do it as expense, transfer? If you know what I mean.

**Reply by M1ck0** [reply]
So, you mean I'd have 2 transactions. One for base amount and one for interest?

Did I get it right?

But even so, main amount should be reduces by base amount, not by interest, as the interest payment doesn't reduce amount of the base loan amount.

**Comment by M1ck0** [comment]
Anyone?
